cfme-versions

```diff
@@ -1,9 +0,0 @@
-#!/usr/bin/env ruby --disable-gems
```

Comment:
Why?  There is no reason for a directory structure in this gem aside from a `test` dir, which I don't (didn't) include in the `.gempsec` on purpose.